src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php

```diff
@@ -87,24 +88,23 @@ public function prepend(ContainerBuilder $container)
             $container->prependExtensionConfig(
                 'fos_rest',
                 [
-                    'routing_loader' => [
-                        'default_format' => 'json',
-                    ],
                     'exception' => [
                         'enabled' => true,
+                        'map_exception_codes' => true,
                         'codes' => [
                             MissingParameterException::class => 400,
                             InvalidHashException::class => 409,
                             ObjectNotSupportedException::class => 406,
                             InvalidFilterTypeOptionsException::class => 400,
                         ],
+                        'exception_listener' => false,
+                        'serialize_exceptions' => false,
+                        'flatten_exception_format' => 'legacy',
                     ],
                     'serializer' => [
                         'serialize_null' => true,
                     ],
-                    'service' => [
-                        'templating' => 'twig',
-                    ],
+                    'routing_loader' => false,
```

Comment:
Why does this need to be set? Isn't false the default value of the route bundle anyway?

Reply:
Not in FOSRestBundle 2.8 which we still support when the project is not updated to symfony 5.